server/pbenchinacan/load_keycloak.sh

```diff
@@ -65,10 +65,45 @@ else
   echo "Created ${REALM} realm"
 fi
 
+# Create a client scope with custom mapper that a functional test
+# user can include in the OIDC token request over a REST call.
+# This will instruct Keycloak to include the CLIENT_ID (pbench-dashboard)
+# when someone request a token Over a rest API using a CLIENT_ID.
```

Comment:
Most of these references appear to be using `CLIENT_ID` as a literal, whereas I think you're intending to be using it as a placeholder for `pbench-dashboard`. A common notational convention like `<client_id>` might help here, or even simply using `pbench-dashboard` as a literal, since that's how the JSON body below is constructed.